spec/rocket_chat/room_spec.rb

```diff
@@ -3,5 +3,7 @@
 require 'spec_helper'
 
 describe RocketChat::Room do
-  pending 'Add some specs for RocketChat::Room'
+  it 'RocketChat::Room spec', pending: 'Add some specs for RocketChat::Room' do
```

Comment:
Not sure why you've changed this? If it's for linter reasons, the previous form was just fine?!

Reply:
Yes, RSpec/PendingWithoutReason was raised. Rubocop in actions is not using rubocop.yml from project root, and pipeline was failing because of this